hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/ReferenceDataService.kt

```diff
@@ -18,29 +19,29 @@ class ReferenceDataService(
   @Value("\${search.detailed.max-results}") private val maxSearchResults: Int = 200,
   @Value("\${search.detailed.timeout-seconds}") private val searchTimeoutSeconds: Long = 10L,
 ) {
+  // we have implemented a cache loader in CacheConfig so that will call the findReferenceData method below
+  @Cacheable(cacheNames = ["referenceData"], key = "#attribute")
+  fun findReferenceDataCached(attribute: ReferenceDataAttribute) = ReferenceDataResponse()
```

Comment:
this is odd - but because we are using a loading cache the implementation will never be called